react-client/src/routes/CreateTutorial.js

```diff
@@ -29,50 +34,129 @@ export default class CreateTutorial extends Component {
     detailTextCounter: null
   }
 
-  createTut() {
+  createTut = () => {
     this.setState({
-      createTutorial: true
+      createTutorial: !this.state.createTutorial
     })
   }
 
-  createSection() {
+  createSection = (e, index, type) => {
     const tutorialData = { ...this.state.tutorialData }
-    tutorialData.sections.push(sectionsDataModel)
-    this.setState({
-      tutorialData
-    })
+    switch (type) {
+      case "create":
+        tutorialData.sections.push(sectionsDataModel)
+        this.setState({
+          tutorialData
+        })
+        break
```

Comment:
Missing ';' ;)

Reply:
This comment applies to all break statements without ';' too XD

Reply:
Not necessarily, my prettier is configured to remove semi-colons where they aren't needed.